common/sgx/eeid.c

```diff
@@ -614,7 +647,7 @@ static oe_result_t _hton_uint64_t(
     return OE_OK;
 }
 
-static oe_result_t _ntoh_uint64_t(
+oe_result_t ntoh_uint64_t(
```

Comment:
If it's not static, then it should have the oe prefix in the function name.
Also, the function name shouldn't end in _t which is reserved for types.
hence I'd recommend `oe_ntoh_uint64`

Reply:
Fixed, thanks!